src/styles/stars.css

```
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
```

Add this, so when your stars--modificator is changed in html-file - you could see actual rating change on your web-page

---

Got this, so i deleted this modificators in vain)